app/core/Router.coffee

```diff
@@ -212,6 +212,8 @@ module.exports = class CocoRouter extends Backbone.Router
     'seen': go('HomeView')
     'SEEN': go('HomeView')
 
+    'students/ranking/:map': go('courses/StudentRankingView')
```

Comment:
Do we need the `:map`?
It doesn't seem to be used in the StudentRankingView.

Reply:
changed to `'students/ranking/:courseID?:courseInstanceID': go('courses/StudentRankingView')` on the next commit.

Reply:
It didn't look like it needed any query params?

Reply:
the parameters are `courseID `and `courseInstanceID`, maybe they are not being declared properly, but are being used.